packages/hub/prisma/schema.prisma

```diff
@@ -222,11 +222,23 @@ model ModelRevision {
   exports               ModelExport[]
 
   // required by Prisma, but unused since `model` field should point at the same entity
-  currentRevisionModel Model? @relation("CurrentRevision")
+  currentRevisionModel Model?               @relation("CurrentRevision")
+  builds               ModelRevisionBuild[]
 
   @@index([modelId])
 }
 
+model ModelRevisionBuild {
+  id              String        @id @default(cuid())
+  createdAt       DateTime      @default(now())
+  modelRevision   ModelRevision @relation(fields: [modelRevisionId], references: [id], onDelete: Cascade)
+  modelRevisionId String
+  runtime         Float
```

Comment:
Can we rename this, or convert to camel case `runTime`, at least? Maybe `runSeconds`.

Too many ways to misunderstand what this means, otherwise.